spec/unit/chef_fs/parallelizer_spec.rb

```diff
@@ -405,7 +405,7 @@ def parallelize(inputs, options = {}, &block)
         expect do
           b = enum.enum_for(:each)
           b.next
-        end.to raise_error
+        end.to raise_error(RuntimeError, "each() called on parallel enumerable twice simultaneously! Bad mojo")
```

Comment:
Expecting `RuntimeError` here seemed a bit too generic so I added the test message.